README.md

```diff
@@ -161,7 +133,7 @@ To build the app from source:
 Squad Leader leverages the following as submodules of the squad-leader-android repository. 
 
 * [military-apps-library-java](https://github.com/Esri/military-apps-library-java) 
-* [aFileChooser](https://github.com/iPaulPro/aFileChooser)
+* [aFileChooser](https://github.com/garys-esri/aFileChooser)
  
 If you should want to update to the latest commit instead of the commit used by the squad-leader-android commit you're using, you can open a GitHub shell in squad-leader-android and run the following:
 
```

Comment:
This version might work better for multiple submodules: `> git submodule foreach git pull origin master`